### PR TITLE
Add the possibility to auto-assign orders from a store

### PR DESCRIPTION
**Pull request by Atala**
auto-assign-to-rider

### CODE REVIEW

File: src/Action/TaskList/SetItems.php [hunk 1]
```diff
@@ -17,45 +17,27 @@
...
-    }
+    {}
 
     public function __invoke(Request $request)
     {
         $date = new \DateTime($request->get('date'));
         $user = $this->userManager->findUserByUsername($request->get('username'));
 
-        $taskList = $this->getTaskList($date, $user);
+        $taskList = $this->taskListManager->getTaskListForUser($date, $user);
```
Comment (r0xsh) [inline src/Action/TaskList/SetItems.php:32]: ~Maybe the `$date` should be the date of the delivery ?~

Comment (r0xsh) [reply src/Action/TaskList/SetItems.php:32]: ~Like what happen if i create a delivery for the next week ?~

**Review by r0xsh — APPROVED** [review]
🚀

File: src/Action/TaskList/SetItems.php (hunk 1 above)
Comment (Atala) [reply src/Action/TaskList/SetItems.php:32]: mmm it is an unrelated change sorry
here we are in the endpoint that assigns task when on the dispatch panel so the $date is a parameter we passed (the date we are showing in the dispatch)

**Pull request merged**